Cargo.toml

```diff
@@ -20,13 +20,17 @@ slab = ["dep:slab_allocator"]
 buddy = ["dep:buddy_system_allocator"]
 
 allocator_api = []
+64G = []
```

Comment:
Can we use a more appropriate name? And maybe we need to specify that the `64G` belongs to `bitmap allocator`(The max pool size in `tlsf` is 8G)